lib/widgets/release_item.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'package:git_touch/models/theme.dart';
+import 'package:git_touch/utils/utils.dart';
+import 'package:git_touch/widgets/avatar.dart';
+import 'package:git_touch/widgets/link.dart';
+import 'package:provider/provider.dart';
+import 'package:timeago/timeago.dart' as timeago;
+
+class ReleaseItem extends StatelessWidget {
+  final String login;
+  final DateTime publishedAt;
+  final String downloadUrl;
+  final String name;
+  final String avatarUrl;
+  final String tagName;
+
+  ReleaseItem(
+      {@required this.login,
+      @required this.publishedAt,
+      @required this.downloadUrl,
+      @required this.name,
+      @required this.tagName,
+      @required this.avatarUrl});
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Provider.of<ThemeModel>(context);
+    return Link(
+      url: downloadUrl,
```

Comment:
Would it be better to go to the release detail web page? For example: https://github.com/git-touch/git-touch/releases/tag/v1.10.1

Reply:
Alright. Sounds better until we can display all the assets so the user knows what they're downloading. Or should I use `ListTile` which expands to display the assets and then use downloadUrl for that?